pkg/broker/types.go

```diff
@@ -304,4 +304,12 @@ type SubscriberDAO interface {
 	SetState(id string, state apb.JobState) (string, error)
 	GetServiceInstance(id string) (*apb.ServiceInstance, error)
 	DeleteServiceInstance(id string) error
+	GetBindInstance(id string) (*apb.BindInstance, error)
+	DeleteBindInstance(id string) error
+	SetServiceInstance(id string, serviceInstance *apb.ServiceInstance) error
```

Comment:
I'm not a fan of the SubscriberDAO. It feels weird to me. But that's for a different day.

Reply:
As this is adding more and more methods, why don't we just make DAO now that DAO is an interface itself?

We either have or easily can create a mock for it as well.

Reply:
Yep no issue with changing that as we already have the DAO interface